packages/flake-module.nix

```diff
@@ -26,23 +26,24 @@
         hardware-scan = callPackage ./hardware-scan { };
         doc = callPackage ../docs {
           revision = lib.strings.fileContents ../.version;
-          # options = ;
-          # TODO Add the options in from the self.nixosModules
-          # The below is not needed anymore to setoptions
-          #
-          # options = let
-          #           cfg = nixpkgs.lib.nixosSystem {
-          #             inherit system;
-          #             modules =
-          #               lib.ghaf.modules
-          #               ++ [
-          #                 jetpack-nixos.nixosModules.default
-          #                 microvm.nixosModules.host
-          #                 lanzaboote.nixosModules.lanzaboote
-          #               ];
-          #           };
-          #         in
-          #           cfg.options;
+          options =
+            let
+              cfg = lib.nixosSystem {
+                # derived from targets/laptop/laptop-configuration-builder.nix + lenovo-x1-carbon-gen10
```

Comment:
Ideally we would expose a nixos module for `lenovo-x1-carbon-gen10` instead of what we are currently doing.